website/src/models/task.js

```diff
@@ -50,10 +50,14 @@ var checklist = [{
 
 var DailySchema = new Schema(
   _.defaults({
-    type: {type:String, 'default': 'daily'},
+    type: {type: String, 'default': 'daily'},
+    //TODO: Cleaner to store interval as enum instead of str?
+    frequency: {type: String, 'default': 'weekly'}, // 'daily', 'weekly'
```

Comment:
Here's an example of an enum: https://github.com/HabitRPG/habitrpg/blob/develop/website/src/models/coupon.js#L10